transaction-service-api/src/main/java/pl/adamboguszewski/transaction/service/api/exception/LogMessage.java

```diff
@@ -0,0 +1,8 @@
+package pl.adamboguszewski.transaction.service.api.exception;
+
+public interface LogMessage {
```

Comment:
It's not a class, its interface which is defining class behaviour so it should be LoggingMessage https://howtodoinjava.com/java/basics/java-naming-conventions/
but its not message either, its an exception